railgun/hack/generators/controllers/networking/main.go

```diff
@@ -20,9 +20,10 @@ const (
 	netv1beta1 = "k8s.io/api/networking/v1beta1"
 	extv1beta1 = "k8s.io/api/extensions/v1beta1"
 
-	kongv1       = "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1"
-	kongv1alpha1 = "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1alpha1"
-	kongv1beta1  = "github.com/kong/kubernetes-ingress-controller/railgun/api/configuration/v1beta1"
+	kongv1          = "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1"
+	kongv1alpha1    = "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1alpha1"
+	kongv1beta1     = "github.com/kong/kubernetes-ingress-controller/railgun/api/configuration/v1beta1"
+	knativev1alpha1 = "knative.dev/networking/pkg/apis/networking/v1alpha1"
```

Comment:
We need to double check that `gofmt` actually would leave this import in this order, I think it will organize it _before_ the kong entries and we should reflect that in this template so that the generator is not producing files that `gofmt` and `goimports` will want to overwrite constantly.

Reply:
I believe it is pretty clear that a new refer is added.